app/robots/iCubGenova04/oculusConfig.ini

```diff
@@ -21,19 +21,19 @@ robot                   icub
 [include LEFT_HAND_RETARGETING "leftHandRetargetingParams.ini"]
 [include RIGHT_HAND_RETARGETING "rightHandRetargetingParams.ini"]
 
-## Comment the following group if you want use the virtualizer
-# [AXIS]
-# deadzone                      0.3
-# fullscale                     1.0
-# scale_X                       5.0
-# scale_Y                       5.0
-# use_lef                       1
-
 [OCULUS]
 root_frame_name                 mobile_base_body_link
 left_hand_frame_name            loculus
 right_hand_frame_name           roculus
 oculusOrientationPort           /oculusOrientation:i
 
+move_icub_using_joypad        1
```

Comment:
In  `app/robots/iCubGenova04/oculusConfig.ini`, this options are under the `[JOYPAD]` group, while here no. It is normal?

Reply:
The idea was to remove the `[JOYPAD]` group and move the params, namely:
```
move_icub_using_joypad        1
deadzone                      0.3
fullscale                     1.0
scale_X                       5.0
scale_Y                       5.0
use_left                      1
```
into the `[OCULUS]` group

Reply:
In 6d848a4 I fixed the `oculusConfig.ini` for icubGazeboSim